ci-operator/config/openshift/assisted-installer/openshift-assisted-installer-release-ocm-2.6.yaml

```diff
@@ -152,6 +160,15 @@ tests:
         SERVICE_BASE_REF=release-ocm-2.6
         NUM_MASTERS=1
     workflow: assisted-baremetal
+- as: e2e-ai-operator-ztp
+  optional: true
```

Comment:
for 2.6 branch I think it should be a required job

Reply:
done